lib/themes/dosomething/paraneue_dosomething/templates/campaign/partials/campaign-headings.tpl.php

```diff
@@ -3,6 +3,6 @@
 
 <?php if (isset($campaign->end_date) && $campaign->status != 'closed'): ?>
   <p class="header__date">
-    <?php print t('Ends @end_date', ['@end_date' => date('F d', $campaign->end_date)]); ?>
+    <?php print t('Ends ' . $campaign_end_date); ?>
```

Comment:
This is incorrect. It will generate all possible combinations of this string in the translation interface. Please refer tot the original code, just don't use `date` function.